tests/CMakeLists.txt

```
@@ -17,41 +16,54 @@ else()
   add_compile_options(-std=c++17 -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wunused -pedantic)
 endif()
 
+if (MSVC OR MSVC_IDE OR MINGW)
+    # must find the RadiumExternal directory. This will be set by RadiumConfig.cmake after install
+    # 
+    # TODO : find a way to that more efficiently
+    find_dependency(cpplocate REQUIRED NO_DEFAULT_PATH)
+    find_dependency(glbinding REQUIRED NO_DEFAULT_PATH)
+    find_dependency(globjects REQUIRED NO_DEFAULT_PATH)
+    find_dependency(tinyply REQUIRED NO_DEFAULT_PATH)
+    set(RadiumExternalDlls_location "")
+    addImportedDir( FROM cpplocate::cpplocate TO RadiumExternalDlls_location) 
+    addImportedDir( FROM glbinding::glbinding TO RadiumExternalDlls_location) 
+    addImportedDir( FROM globjects::globjects TO RadiumExternalDlls_location)
+    addImportedDir( FROM tinyply TO RadiumExternalDlls_location) 
```

The use of tinyply is garded by the option `RADIUM_IO_TINYPLY`, so I suggest:
```suggestion
    if (RADIUM_IO_TINYPLY)
        addImportedDir( FROM tinyply TO RadiumExternalDlls_location) 
    endif ()
```

---

Should be done. If I didn't mixup with rebases.